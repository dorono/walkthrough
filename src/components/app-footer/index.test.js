import React from 'react';
import {shallow} from 'enzyme';
import AppFooter from './index';

const defaultProps = {
    apiConfig: {},
};

describe('AppFooter', () => {
    it('should render without errors', () => {
        shallow(<AppFooter {...defaultProps} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<AppFooter {...defaultProps} />)).toMatchSnapshot();
    });
});
