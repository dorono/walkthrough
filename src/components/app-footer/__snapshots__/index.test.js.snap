// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter should match snapshot 1`] = `
<footer
  className="root"
>
  <span>
    Â© 
    2019
     Factom, Inc.
  </span>
  <Monospaced>
    UI: 
     / API: 
    Loading
  </Monospaced>
</footer>
`;
